equinix_metal/.gitlab-ci.yml

```diff
@@ -16,10 +16,10 @@ stages:
 
 pytest-3.7:
   extends: .pytest
-  image: python:3.11-alpine
+  image: python:3.7-alpine
```

Comment:
Is this a generated file or are we using GitLab CI somehow?

Reply:
It's generated by OAG, well spotted. I'll try to get rid of it.